pages/new-collective-page.js

```diff
@@ -132,6 +141,7 @@ class NewCollectivePage extends React.Component {
     }
 
     const collective = data && data.Collective;
+    const confirmedStatus = this.getStatus(status, collective);
```

Comment:
I'm not sure about this part, it's making the code complicated, could you propose an alternative?